### PR TITLE
feat(docs):  widget proofread wrap-up

### PR DESCRIPTION
**Pull request by vwheeler63**
This update contains minor improvements to already-proofread docs during a time when they were pending in a PR.  So they were saved for after the pending PRs containing them were merged to avoid conflicts.

NOTE:  As of 15:56 4-Dec-2024 CET, I found a problem with `display.rst` (directives require a blank line after them to be parsed correctly — there were 2 new `.. code-block:: c` directives that needed the blank line — without the blank line, the code block gets omitted from Sphinx output) and fixed it with some minor re-wording for clarity.  Since this PR had no approvals yet, I added it herein, since it fit the same category:  fixes/refinements to already-proofread docs.

Note to @kisvegabor :  I think you needed my updates to `build.py` so you could generate docs locally and test the results.  I will submit that today.

Note for both of you -- the updated generated forms of all the files submitted are visible [here](http://crystal-clear-research.com/for_gabor/demo/docs.lvgl.io/master/details/widgets/index.html).

cc:  @kisvegabor @liamHowatt 

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.  Done.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.  n/a
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.  n/a
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).  n/a
- Run `scripts/code-format.py` (`astyle v3.4.12` needs to installed by running `cd scripts; ./install_astyle.sh`) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).  Done.
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.  Done.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.  Done.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

**Review by kisvegabor — DISMISSED** [review]
Thank you! Feel free to send P to update `build.py` is it needs to be updated :slightly_smiling_face:

**Comment by vwheeler63** [comment]
> Thank you! Feel free to send P to update `build.py` is it needs to be updated 🙂

What does "send P" mean?

**Comment by vwheeler63** [comment]
Hi, Gábor!  ( @kisvegabor )

The recent commit of `bar.rst` caused a conflict.  I see the content changes are good and I understand why you made them.  I re-based this branch to resolve, but unfortunately, it caused your approval to be invalidated.  No additional changes were made to this branch other than incorporating your changes to `bar.rst` into it (via resolving the conflict on my local workstation).

File: docs/details/main-components/display.rst [hunk 1]
```diff
@@ -387,11 +387,11 @@ LVGL supports rotation of the display in 90 degree increments.
...
 LVGL will swap the horizontal and vertical resolutions internally
-according to the set degree, however will not perform the actual rotation.
+according to the set degree. When changing the rotation
 
 When changing the rotation :cpp:enumerator:`LV_EVENT_SIZE_CHANGED` event is
 emitted (for any part of your system that has "subscribed" to this event) to
-allow reconfiguring of the hardware. In lack of hardware display rotation support
+In lack of hardware display rotation support
 :cpp:func:`lv_draw_sw_rotate` can be used to rotate the buffer in the
 :ref:`flush_callback`.
```
Comment (liamHowatt) [inline docs/details/main-components/display.rst]: Incomplete sentence

File: docs/details/widgets/arc.rst [hunk 2]
```diff
@@ -104,7 +104,7 @@ In other words, the angle and value settings are independent.  You should
 exclusively use one or the other of the two methods.  Mixing the two could
 result in unintended behavior.
 
-To make the arc non-adjustable by external input, remove the style of the knob and
+To make the arc non-adjustable, remove the style of the knob and make
 make the Widget non-clickable:
```
Comment (liamHowatt) [inline docs/details/widgets/arc.rst]:
```suggestion
To make the arc non-adjustable, remove the style of the knob and
make the Widget non-clickable:
```

File: docs/details/widgets/bar.rst [hunk 3]
```diff
@@ -37,8 +37,8 @@ Usage
 Orientation and size
 --------------------
 
-- for orientation, width and height, simply set width and height properties;
-- :cpp:expr:`lv_bar_set_orientation(bar, orientation)` to override orientation
+- for orientation, width and height, simply set width and height style properties;
+- :cpp:expr:`lv_bar_set_orientation(slider, orientation)` to override orientation
```
Comment (liamHowatt) [inline docs/details/widgets/bar.rst]: I think this should stay as `bar`

File: docs/details/widgets/calendar.rst [hunk 4]
```diff
@@ -31,9 +31,11 @@ Parts and Styles
...
 Widget under the hood to arrange the days into a matrix.
 
-- :cpp:enumerator:`LV_PART_MAIN` Calendar background. Uses all the background-related style properties.
-- :cpp:enumerator:`LV_PART_ITEMS` Refers to dates and day names. Button matrix control flags are set to differentiate the
-  buttons and a custom drawer event is added modify the properties of the buttons as follows:
+- :cpp:enumerator:`LV_PART_MAIN` Calendar background. Uses the :ref:`typical
+  background style properties <typical bg props>`.
+- :cpp:enumerator:`LV_PART_ITEMS` Refers to dates and day names. Button matrix
+  control flags are set to differentiate the buttons and a custom drawer event is
+  added modify the properties of the buttons as follows:
```
Comment (liamHowatt) [inline docs/details/widgets/calendar.rst]:
```suggestion
  added to modify the properties of the buttons as follows:
```

**Review by liamHowatt — COMMENTED** [review]
Nice!

File: docs/details/main-components/display.rst (hunk 1 above)
Comment (vwheeler63) [reply docs/details/main-components/display.rst:396]: Dang -- conflict resolution error on my part.  :-(

Comment (vwheeler63) [reply docs/details/main-components/display.rst]: Dang -- conflict resolution error on my part.

File: docs/details/widgets/bar.rst (hunk 3 above)
Comment (vwheeler63) [reply docs/details/widgets/bar.rst]: Wow -- must have been a copy/paste casualty.  Good catch!

**Comment by vwheeler63** [comment]
@liamHowatt @kisvegabor All fixed.

**Review by liamHowatt — APPROVED** [review]
Thanks Victor!

**Review by kisvegabor — APPROVED** [review]
Thank you!

**Comment by vwheeler63** [comment]
> Thank you!

And thank you!  😄

**Pull request merged**